components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -217,7 +227,7 @@
             </plugin>
             <plugin>
                 <artifactId>maven-war-plugin</artifactId>
-                <version>2.2</version>
+                <version>3.3.1</version>
```

Comment:
define this at the parent pom file under versions Eg: ${maven.war.plugin.version}. Fix this in all places